docsite/docs/testing.md

```diff
@@ -0,0 +1,173 @@
+---
```

Comment:
Should we add this part also to the readme?

Reply:
I think it would be a nice addition. Because when I first checked RX_Command I didn't really know how to test as I was also new to testing streams. Then I checked how it was done in main repo and wrote my test cases. So a brief introduction could help newbies like me. 😄

Reply:
yep that would be good